src/vellum/workflows/edges/__init__.py

```diff
@@ -1,5 +1,7 @@
 from .edge import Edge
+from .trigger_edge import TriggerEdge
```

Comment:
[blocking] This might make sense, but I'd like to rule out that our current Edge can't be used first. What do we currently perceive as the blocker preventing `Edge` from being able to handle `Trigger`'s as a source?